packages/nimble-components/src/anchor/index.ts

```diff
@@ -0,0 +1,53 @@
+import { attr } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    Anchor as FoundationAnchor,
+    applyMixins,
+    DelegatesARIALink,
+    AnchorOptions
+} from '@microsoft/fast-foundation';
+import { styles } from './styles';
+import { template } from './template';
+import type { AnchorAppearance } from './types';
+
+export { FoundationAnchor };
```

Comment:
The commit associated with this change says this is to support the Angular Wrapper but that conceptually seems wrong.. the type we expose is Nimble Anchor, why would the angular wrapper need to know about our base class?

Reply:
I created a common base

Reply:
I'm using a common base class for the Anchor and AnchorButton wrappers, since they share all the anchor attributes (e.g. href, etc):
```
export class NimbleLinkBase<T extends FoundationAnchor> {
    ...
    public constructor(protected readonly renderer: Renderer2, protected readonly elementRef: ElementRef<T>) {}
}
```
I initially tried importing `FoundationAnchor` from FAST, but I got an error saying that we should re-export Nimble dependencies from Nimble and import from there instead.

Reply:
if it's needed then yea it should be exported from nimble. I don't think it makes sense to export it from the Anchor file as that file does not represent the base class but an implementation. Also we don't want to expose implementation details of our components outside nimble generally.

Maybe make an `/src/anchor-base/index.ts` that contains
```ts
import { Anchor as FoundationAnchor } from '@microsoft/fast-foundation';
export class AnchorBase extends FoundationAnchor {}
```

And then make sure our anchor and anchor-button extend that.

Reply:
Done